data/topic-schema/src/main/resources/net/corda/schema/VirtualNode.yaml

```diff
@@ -82,4 +82,17 @@ topics:
       - db
     producers:
       - rest
-    config:
\ No newline at end of file
+    config:
+  VirtualNodeOperationStatusTopic:
+    name: virtual.node.operation.status
+    consumers:
+      - db
+    producers:
+      - rest
```

Comment:
Are these not the wrong way round?

Reply:
I think producer needs to be both `rest` and `db` and consumer `rest`

Reply:
fixed